docs/hugo/content/reference/breaking-changes.md

```diff
@@ -0,0 +1,434 @@
+# Breaking Changes in beta.4
+
+In the 'beta.4' release of Azure Service Operator (ASO) we are pivoting to using the Azure Swagger API Specifications as our sole source of truth for our code generator.
+
+This change brings with in a significant improvement in fidelity - the code we generate is now much closer to what the Azure Swagger API Specifications describe.
+
+Unfortunately, this change brings with it a few breaking changes, listed here.
+
+These changes fall into a few categories:
+
+**Unused**: Properties that were previously generated but actually had no function have been removed.
+
+Examples include:
+
+* `Location` on subresources that inherit their actual location from their parent resource
+* `Tags` on resources that don't support tags
+
+If your resource Spec was using one of these properties, no action is required; the new release of ASO will silently ignore them.
+
+**Status Only**: Properties that cannot set by the end user on a resource spec. Typically these are actually Status properties included in the Spec in error.
+
+Examples include:
+
+* Identity.UserAssignedIdentities
+
+If your resource Spec was using one of these properties, no action is required; the new release of ASO will silently ignore them.
```

Comment:
This comment doesn't seem to make sense in context, as you're talking about `Status Only` changes, so `Spec` is not involved?

Reply:
It's talking about properties that only make sense on `Status` that were incorrectly being included on `Spec`. I'll update the wording.